### PR TITLE
Updating App Insights documentation 

### PR DESCRIPTION
**Pull request by MichaelMendix**
Updated name "Product Insights" to "App Insights"
Remove deprecated settings instructions for feature we don't use anymore 
Updated screenshot to reflect removal of deprecated settings feature

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@Luyao-Zhang-1 are you able to pick this up? I verified that it can be published immediately.

File: content/en/docs/developerportal/collaborate/product-insights.md [hunk 1]
```diff
@@ -118,14 +118,6 @@ You can collect test data using two methods:
...
 * Select the **Enable Mendix SSO test users"** check box on the **Settings** tab of **Survey Overview**, which marks the app team members as testers. Note that if an app team member is not authenticated through Mendix SSO, this setting does not work for them. The survey widget will not show **[TEST_MODE]** on the UI.
 
-### 2.3 Product Insights Settings {#product-insights-settings}
-
-When you click the settings icon on [Survey Overview](#survey-overview), **Product Insights Settings** opens.
-
-{{< figure src="/attachments/developerportal/collaborate/product-insights/settings-icon.png" >}}
-
-Here you can view the existing API key, or generate an API key if there is not one available. You need to use the API key as input when you configure the App Insights – Mini-Surveys module in Studio Pro to implement the mini-survey. You need only one API key per app.
-
```
Comment (Luyao-Zhang-1) [inline content/en/docs/developerportal/collaborate/product-insights.md:128]: Then how can we get to **Product Insights Settings** to view/generate an API key? @MichaelMendix

Comment (MichaelMendix) [reply content/en/docs/developerportal/collaborate/product-insights.md:128]: Ahhh, 😲 good question, thank you for finding this

So we moved the API Key Generate to the Survey Settings Tab Page (2.1.3 Settings Tab https://docs.mendix.com/developerportal/collaborate/product-insights/#settings)

I think API Key Generate has to be mentioned in  (2.1.3 Settings Tab https://docs.mendix.com/developerportal/collaborate/product-insights/#settings)

Comment (MichaelMendix) [reply content/en/docs/developerportal/collaborate/product-insights.md:128]: @Luyao-Zhang-1 

Okay I have adjusted the documentation and also added a few more details/sections 😊

**Pull request merged**